sectionctl.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/willabides/kongplete"
 )
 
+//go:generate go-winres make --product-version=git-tag --file-version=git-tag
```

Comment:
What is this line doing?

Reply:
this is windows-specific binary metadata that you can generate via `go generate`